.github/workflows/ci.yml

```diff
@@ -58,6 +58,7 @@ jobs:
           poetry config http-basic.epc-power aws $CODEARTIFACT_AUTH_TOKEN
           poetry config repositories.third-party https://epc-power-us-${{ secrets.AWS_ACCOUNT_ID }}.d.codeartifact.us-east-2.amazonaws.com/pypi/third-party/
           poetry config http-basic.third-party aws $CODEARTIFACT_AUTH_TOKEN
+          poetry config installer.max-workers 1
```

Comment:
Add this comment so that we can remember why we put in the line and revisit later.
```suggestion
          # https://github.com/python-poetry/poetry/issues/7611
          poetry config installer.max-workers 1
```

Reply:
This comment still hasn't been added.

Reply:
It does seem like without it; this project won't have access to the functionality provided by poetry dynamic versioning.

Reply:
Added.